org.gnome.Evince.json

```
@@ -99,8 +82,8 @@
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://poppler.freedesktop.org/poppler-0.74.0.tar.xz",
-                    "sha256": "92e09fd3302567fd36146b36bb707db43ce436e8841219025a82ea9fb0076b2f"
+                    "url": "https://poppler.freedesktop.org/poppler-0.77.0.tar.xz",
```

Poppler has releases about once per month, so this dependency is already out of date.

```suggestion
                    "url": "https://poppler.freedesktop.org/poppler-0.79.0.tar.xz",
```